src/Cli/Commands/SeoCheckCommand.php

```diff
@@ -0,0 +1,95 @@
+<?php
+namespace Frickelbruder\KickOff\Cli\Commands;
+
+use Frickelbruder\KickOff\App\Application;
+use Frickelbruder\KickOff\App\KickOff;
+use Frickelbruder\KickOff\Log\Listener\JunitLogListener;
+use Frickelbruder\KickOff\Log\Listener\ListenerFactory;
+use Frickelbruder\KickOff\Log\Logger;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class SeoCheckCommand extends Command {
+
+    /**
+     * @var Logger
+     */
+    protected $logger = null;
+
+    /**
+     * @var KickOff
+     */
+    private $mainApplication;
+
+
+    private $logs = array(
+        array('name' => 'junit-file', 'shortcut' => 'j', 'mode' => InputOption::VALUE_OPTIONAL, 'description' => 'path to a junit compatible log file'),
```

Comment:
Are those needed? If the command is meant only to have a quick feedback we can remove that I guess